HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -266,6 +266,28 @@ class HyprMXAdapter : PartnerAdapter {
         }
     }
 
+
+    /**
+     * Set HyprMX user's consent value using a boolean.
+     * This is for publishers to manually set the consent status.
+     * This uses CONSENT_GIVEN for true and CONSENT_DECLINED for false.
+     *
+     * @param context a context that will be passed to the SharedPreferences to set the user consent.
+     * @param applies True if GDPR applies, false otherwise.
+     * @param consented whether or not the user has consented.
+     */
+    fun setGdpr(
+        context: Context,
+        applies: Boolean?,
+        consented: Boolean,
+    ) {
+        setGdpr(
+            context,
+            applies,
+            if (consented) GdprConsentStatus.GDPR_CONSENT_GRANTED else GdprConsentStatus.GDPR_CONSENT_DENIED,
```

Comment:
so technically applies=null = unknown, right? (though it likely doesn't matter; most of this is going away)